managed/ui/src/pages/Toaster.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import '../app/stylesheets/Toaster.scss';
+
+
+const Toaster = (props) => {
+  const { toast, onDismissClick} = props;
```

Comment:
Please maintain the indentation across all the files

Reply:
Noted